mopidy_spotify/playlists.py

```diff
@@ -47,7 +54,15 @@ def _get_playlist(self, uri, as_items=False):
         )
 
     def refresh(self):
-        pass  # TODO: Clear/invalidate all caches on refresh.
+        with utils.time_logger("Refresh Playlists", logging.INFO):
```

Comment:
Replace "Refresh playlists" with "playlists.refresh()" for consistency with the other time loggers?